### DISCUSSION TITLE
Is the documentation hosted on a website?

### DISCUSSION THREAD

**Discussion by ordinary-slim** [multiphenics/multiphenicsx#36] [General]
Is the documentation hosted on a website? That would be practical but it is not essential. Table of contents looks a bit weird, not sure if it is avoidable:

![image](https://github.com/user-attachments/assets/901831e7-373d-41f6-900f-65d510ea0ed9)

Maybe changing the sphinx theme would fix

**Comment by francesco-ballarin** [comment]
Not currently hosted anywhere. Honestly it's not a high priority for me. What I dislike about having it on a website is that once we do that we'd have to host documentations of several versions (just like dolfinx does: e.g. dolfinx v0.9, v.0.8, etc), and that becomes a lot of work for very little benefit.
